2024-09-23 10:19:48,759 INFO    StreamThr :14580 [internal.py:wandb_internal():85] W&B internal server running at pid: 14580, started at: 2024-09-23 10:19:48.759096
2024-09-23 10:19:48,761 DEBUG   HandlerThread:14580 [handler.py:handle_request():158] handle_request: status
2024-09-23 10:19:48,762 INFO    WriterThread:14580 [datastore.py:open_for_write():87] open: E:\DeepPurposePlusPlus\wandb\run-20240923_101948-40prff02\run-40prff02.wandb
2024-09-23 10:19:48,764 DEBUG   SenderThread:14580 [sender.py:send():379] send: header
2024-09-23 10:19:48,791 DEBUG   SenderThread:14580 [sender.py:send():379] send: run
2024-09-23 10:19:49,186 INFO    SenderThread:14580 [dir_watcher.py:__init__():211] watching files in: E:\DeepPurposePlusPlus\wandb\run-20240923_101948-40prff02\files
2024-09-23 10:19:49,191 DEBUG   HandlerThread:14580 [handler.py:handle_request():158] handle_request: check_version
2024-09-23 10:19:49,194 INFO    SenderThread:14580 [sender.py:_start_run_threads():1188] run started: 40prff02 with start time 1727079588.760193
2024-09-23 10:19:49,194 DEBUG   SenderThread:14580 [sender.py:send_request():406] send_request: check_version
2024-09-23 10:19:49,274 DEBUG   HandlerThread:14580 [handler.py:handle_request():158] handle_request: run_start
2024-09-23 10:19:49,762 DEBUG   HandlerThread:14580 [system_info.py:__init__():26] System info init
2024-09-23 10:19:49,762 DEBUG   HandlerThread:14580 [system_info.py:__init__():41] System info init done
2024-09-23 10:19:49,762 INFO    HandlerThread:14580 [system_monitor.py:start():194] Starting system monitor
2024-09-23 10:19:49,762 INFO    SystemMonitor:14580 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-09-23 10:19:49,762 INFO    HandlerThread:14580 [system_monitor.py:probe():214] Collecting system info
2024-09-23 10:19:49,767 INFO    SystemMonitor:14580 [interfaces.py:start():188] Started cpu monitoring
2024-09-23 10:19:49,768 INFO    SystemMonitor:14580 [interfaces.py:start():188] Started disk monitoring
2024-09-23 10:19:49,768 INFO    SystemMonitor:14580 [interfaces.py:start():188] Started gpu monitoring
2024-09-23 10:19:49,775 INFO    SystemMonitor:14580 [interfaces.py:start():188] Started memory monitoring
2024-09-23 10:19:49,785 INFO    SystemMonitor:14580 [interfaces.py:start():188] Started network monitoring
2024-09-23 10:19:49,788 DEBUG   HandlerThread:14580 [system_info.py:probe():152] Probing system
2024-09-23 10:19:49,789 DEBUG   HandlerThread:14580 [system_info.py:_probe_git():137] Probing git
2024-09-23 10:19:49,839 DEBUG   HandlerThread:14580 [system_info.py:_probe_git():145] Probing git done
2024-09-23 10:19:49,839 DEBUG   HandlerThread:14580 [system_info.py:probe():200] Probing system done
2024-09-23 10:19:49,839 DEBUG   HandlerThread:14580 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.9.18', 'heartbeatAt': '2024-09-23T08:19:49.788147', 'startedAt': '2024-09-23T08:19:48.747675', 'docker': None, 'cuda': None, 'args': ('--target_encoding', 'DGL_NeuralFP', '--seed', '100', '--wandb_proj', 'DeepPurposePP', '--lr', '0.00001', '--epochs', '100'), 'state': 'running', 'program': 'E:\\DeepPurposePlusPlus\\train\\yeast_ppi.py', 'codePathLocal': 'train\\yeast_ppi.py', 'codePath': 'train\\yeast_ppi.py', 'git': {'remote': 'https://github.com/jiaqingxie/ProB.git', 'commit': 'ef84d631fe7187f2baec682cfb00061253bd3006'}, 'email': 'jiaxie@student.ethz.ch', 'root': 'E:/DeepPurposePlusPlus', 'host': 'DESKTOP-E5HJ25M', 'username': '嘉庆', 'executable': 'E:\\anaconda3\\envs\\graph\\python.exe', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 2100.0, 'min': 0.0, 'max': 2100.0}, 'cpu_freq_per_core': [{'current': 2100.0, 'min': 0.0, 'max': 2100.0}], 'disk': {'/': {'total': 214.84276962280273, 'used': 75.60845947265625}}, 'gpu': 'NVIDIA GeForce RTX 4070 Laptop GPU', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4070 Laptop GPU', 'memory_total': 8585740288}], 'memory': {'total': 15.745964050292969}}
2024-09-23 10:19:49,839 INFO    HandlerThread:14580 [system_monitor.py:probe():224] Finished collecting system info
2024-09-23 10:19:49,839 INFO    HandlerThread:14580 [system_monitor.py:probe():227] Publishing system info
2024-09-23 10:19:49,839 DEBUG   HandlerThread:14580 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-09-23 10:19:49,840 ERROR   HandlerThread:14580 [system_info.py:_save_conda():223] Error saving conda packages: [WinError 2] 系统找不到指定的文件。
Traceback (most recent call last):
  File "E:\anaconda3\envs\graph\lib\site-packages\wandb\sdk\internal\system\system_info.py", line 216, in _save_conda
    subprocess.call(
  File "E:\anaconda3\envs\graph\lib\subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "E:\anaconda3\envs\graph\lib\subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "E:\anaconda3\envs\graph\lib\subprocess.py", line 1436, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] 系统找不到指定的文件。
2024-09-23 10:19:49,841 DEBUG   HandlerThread:14580 [system_info.py:_save_conda():224] Saving conda packages done
2024-09-23 10:19:49,841 INFO    HandlerThread:14580 [system_monitor.py:probe():229] Finished publishing system info
2024-09-23 10:19:49,849 DEBUG   SenderThread:14580 [sender.py:send():379] send: files
2024-09-23 10:19:49,849 INFO    SenderThread:14580 [sender.py:_save_file():1454] saving file wandb-metadata.json with policy now
2024-09-23 10:19:50,007 DEBUG   HandlerThread:14580 [handler.py:handle_request():158] handle_request: python_packages
2024-09-23 10:19:50,007 DEBUG   SenderThread:14580 [sender.py:send_request():406] send_request: python_packages
2024-09-23 10:19:50,008 DEBUG   HandlerThread:14580 [handler.py:handle_request():158] handle_request: stop_status
2024-09-23 10:19:50,008 DEBUG   SenderThread:14580 [sender.py:send_request():406] send_request: stop_status
2024-09-23 10:19:50,008 DEBUG   HandlerThread:14580 [handler.py:handle_request():158] handle_request: internal_messages
2024-09-23 10:19:50,192 INFO    Thread-16 :14580 [dir_watcher.py:_on_file_created():271] file/dir created: E:\DeepPurposePlusPlus\wandb\run-20240923_101948-40prff02\files\requirements.txt
2024-09-23 10:19:50,193 INFO    Thread-16 :14580 [dir_watcher.py:_on_file_created():271] file/dir created: E:\DeepPurposePlusPlus\wandb\run-20240923_101948-40prff02\files\wandb-metadata.json
2024-09-23 10:19:50,193 INFO    Thread-16 :14580 [dir_watcher.py:_on_file_created():271] file/dir created: E:\DeepPurposePlusPlus\wandb\run-20240923_101948-40prff02\files\conda-environment.yaml
2024-09-23 10:19:50,279 DEBUG   SenderThread:14580 [sender.py:send():379] send: telemetry
2024-09-23 10:19:50,279 DEBUG   SenderThread:14580 [sender.py:send():379] send: config
2024-09-23 10:19:50,387 INFO    wandb-upload_0:14580 [upload_job.py:push():130] Uploaded file C:\Users\嘉庆\AppData\Local\Temp\tmplhpgts2qwandb\dmn615v6-wandb-metadata.json
2024-09-23 10:19:51,198 INFO    Thread-16 :14580 [dir_watcher.py:_on_file_created():271] file/dir created: E:\DeepPurposePlusPlus\wandb\run-20240923_101948-40prff02\files\output.log
2024-09-23 10:19:51,818 ERROR   gpu       :14580 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-23 10:19:51,818 ERROR   gpu       :14580 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-23 10:19:53,213 INFO    Thread-16 :14580 [dir_watcher.py:_on_file_modified():288] file/dir modified: E:\DeepPurposePlusPlus\wandb\run-20240923_101948-40prff02\files\output.log
2024-09-23 10:19:54,507 DEBUG   HandlerThread:14580 [handler.py:handle_request():158] handle_request: status_report
2024-09-23 10:19:56,213 ERROR   gpu       :14580 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-23 10:19:56,215 ERROR   gpu       :14580 [interfaces.py:monitor():142] Failed to sample metric: Unknown Error
2024-09-23 10:19:59,534 DEBUG   HandlerThread:14580 [handler.py:handle_request():158] handle_request: status_report
2024-09-23 10:19:59,827 INFO    cpu       :14580 [interfaces.py:monitor():138] Process cpu has exited.
2024-09-23 10:19:59,827 DEBUG   SystemMonitor:14580 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-09-23 10:19:59,827 DEBUG   SystemMonitor:14580 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-09-23 10:19:59,828 DEBUG   SystemMonitor:14580 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-09-23 10:19:59,828 DEBUG   SenderThread:14580 [sender.py:send():379] send: stats
2024-09-23 10:20:00,372 INFO    MainThread:14580 [internal.py:handle_exit():75] Internal process exited
